### ISSUE TITLE
Global search broken when using vue-good-table in DOM template

### ISSUE THREAD

**Issue by advicepyro** [xaksis/vue-good-table#190]
The table in my project is defined as follows:

```
<vue-good-table
        id="data-table"
        :columns="workerTableColumns"
        :rows="workerTableData"
        :paginate="true"
        style-class="table table-bordered table-striped"
        v-bind:row-style-class="getThisRowClassAndId"
        :global-search="true"
        :sortable="false"
        :per-page="50"
>
```

With 1.19.0 search works in my table. Starting from 1.19.2, search produces an error in the console:

![screen shot 2018-02-14 at 1 26 46 pm](https://user-images.githubusercontent.com/6377466/36221115-13350c3e-118b-11e8-96f4-bac78d690521.png)

**Comment by xaksis** [comment]
@advicepyro thanks for pointing this out, fix coming up shortly

**Comment by xaksis** [comment]
@advicepyro this should now be fixed in 1.19.3

**Comment by jjhesk** [comment]
im on vue 2.5.13 and vue beta.7 and this is still the problem.
